### PR TITLE
Add continuous build

### PR DESCRIPTION
**Pull request by jfarrell468**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: cpp
+script: ./build_tests.sh && ./config_parser_test
+addons:
+  apt:
+    packages:
+    - libboost-all-dev
```
Comment (gregbillock) [inline .travis.yml]: You shouldn't need to install boost.

**Comment by gregbillock** [comment]
Why does the file have a leading dot? Is that the conventional name for travis?

**Comment by jfarrell468** [comment]
Yes, .travis.yml is the magic file name that travis-ci looks for.

Fixed the boost issue. PTAL.

**Comment by gregbillock** [comment]
lgtm

**Pull request merged**